### DISCUSSION TITLE
How can i add Tron Network Provider ?

### DISCUSSION THREAD

**Discussion by deantek** [wevm/wagmi#2150] [Q&A]
I want to be able to connect metamask wallet, tronlink and wallet connect, can i add tron ​​network and will it work?

**Comment by tmm** [comment]
For chains that aren't supported in wagmi yet, you can [build your own.](https://wagmi.sh/react/chains#build-your-own) You are welcome to add the chain to wagmi by opening a PR in the [references repo](https://github.com/wagmi-dev/references) as well.
